mint-choc-chip-skyblade/sshd-rando
# Fixes and tweaks

## What does this PR do?
* Fix logic for Floria Waterfall - Slingshot Cistern Door Frame checks
* Fix hash when starting with multiple of the same item
* Add beta testers to the README
* Fix Bucha being in the wrong state in FFW after getting his reward check
* Set respawn info when starting a new game file to prevent crashes if the player dies before the respawn info can otherwise get set
* Create an autosave when starting a new game file
* Slow down the loftwing speed during the Levias fight
* Change SotH parts to be progressive
  * There are now 4 SotH parts to match the vanilla game
* Add a sword requirement for entering trial gates

## How do you test this changes?
I tested:
* The hash issue was fixed
* Bucha behaves properly at every stage of the slingshot quest and later when Owlan's quest is active
* I tested dying before reloading the first area of a new game file
* I tested loading the autosave created when starting a new game file behaves
* I tested the loftwing speed only slows down in the Thunderhead when the fight is active. The loftwing doesn't slow down before the fight or once it's been defeated
* I tested each stage of the progressive SotH items works as expected
  * I tested starting with all the SotH parts
  * I tested the SotH change with the "skip harp playing" setting
* I tested trying to enter a trial gate without a sword and it did not work
* I tested trying to enter a trial gate with the practice sword and it *did* work
* I tested the logic for the SotH and the trial gates works correctly with the tracker
